### ISSUE TITLE
Variants of humanoid characters don't use their inherited character's ragdoll by default

### ISSUE THREAD

**Issue by Emanthealmighty** [Regalis11/Barotrauma#9089]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When spawning a variant of a humanoid monster, the game will throw an error stating that it couldn't find a ragdoll file for the variant, and then uses the ragdoll file of the character the variant is supposed to inherit it from.

Prior to the Urban Expanses update, this used to not be the case, and the variant would simply use the inherited character's ragdoll without causing any errors.

### Reproduction steps

1. Create a humanoid monster (Could use the "husk" character for this)
2. Create a variant of said humanoid monster, without actually changing anything about it
3. Spawn the variant anywhere in-game
4. Observe how the game throws an error about the character's ragdoll file, and then proceeds to spawn it properly anyway

[Mod that contains a variant of the "Husk" character](https://github.com/Regalis11/Barotrauma/files/8647127/Testmod.zip)

### Bug prevalence

Happens every time I play

### Version

0.17.15.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
This is what the errors will look like, replacing [VARIANTSPECIESNAME] with whatever the variant's species name is:
[RagdollParams] Could not find a ragdoll file that matches the name [VARIANTSPECIESNAME]DefaultRagdoll. Using the default ragdoll.
```

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@615b5f8.

**Comment by Regalis11** [comment]
Seems to have broken preloading character variants, moving back to Review.

![image](https://user-images.githubusercontent.com/14351070/171132715-7869d444-41fa-49dc-98e5-64325f2975e1.png)

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@6659294.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
